questions/qa-backwards-deletion.en.html

```diff
@@ -30,24 +30,82 @@
 
 f.additionalLinks = ''
 </script>
-<script type="text/javascript" src="qa-backwards-deletion/translations.js"> </script>
-<script type="text/javascript" src="../javascript/doc-structure/article-dt.js"> </script>
-<script type="text/javascript" src="../javascript/boilerplate-text/boilerplate-en.js"></script><!--TRANSLATORS must change -en to the subtag for their language!-->
-<script type="text/javascript" src="../javascript/doc-structure/article.js"> </script>
-<script type="text/javascript" src="../javascript/articletoc-html5.js"></script>
-<link rel="stylesheet" href="../style/article-2016.css" type="text/css" />
-<!-- remove the following local css declaration if there is nothing to add -->
-<link rel="stylesheet" href="filename-data/local.css" type="text/css" />
+<script src="qa-backwards-deletion/translations.js"> </script>
+<script src="../javascript/doc-structure/article-dt.js"> </script>
+<script src="../javascript/boilerplate-text/boilerplate-en.js"></script><!--TRANSLATORS must change -en to the subtag for their language!-->
+<script src="../javascript/doc-structure/article-2022.js"> </script>
+<script src="../javascript/articletoc-2022.js"></script>
+<link rel="stylesheet" href="../style/article-2022.css" />
 <link rel="copyright" href="#copyright"/>
-<!--[if lt IE 9]><script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
-
-<style type="text/css">
+<script src="../javascript/prism.js"></script>
+<link rel="stylesheet" href="../style/prism.css">
+<style>
 td.bigtext {
     font-size: 36pt;
     text-align: center;
     font-family: "Khmer", "noto_devanagari", "noto_symbols", "noto_regular", "NoToFu", "Palatino", "Times", "Times New Roman", "Verdana", sans-serif;
 }
+
+button.reset {
+	font-size: 9pt;
+	font-family: monospace;
+}
+
+input.try {
+	font-size: 24pt;
+}
+
+div.try {
+	border:1px solid black;
+	background-color:#ccc;
+}
+
+	.codepoint bdi {
+		line-height: 1em;
+		font-size: 1.25em;
+		padding-inline: 0.25em;
+	}
+	
+	.codepoint img {
+	    height: 2em;
+	    margin-inline: 0.25em;
+        vertical-align: bottom;
+	}
+
+	.uname {
+	    font-family: Arial, monospace;
+		font-size: 0.75em;                    
+		letter-spacing: 0.03em;
+		color: brown;
+	}
```

Comment:
I think this should be in the [shared CSS file](https://github.com/w3c/i18n-drafts/blob/gh-pages/style/article-2022.css), instead of this article.

Reply:
Done